exec/java-exec/src/main/java/org/apache/drill/exec/store/pcapng/Schema.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.pcapng;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Schema {
+  private final static Map<String, PcapngTypes> columns = new LinkedHashMap<>();
```

Comment:
Yes, this is ordered map class, because of this I have a hard-coded order of columns. I will change it because we must arrange the order of columns based on a projection, not on a hard-coded order, now I understand it.